types/usercentrics-browser-ui/package.json

```diff
@@ -0,0 +1,19 @@
+{
+    "private": true,
+    "name": "@types/usercentrics-browser-ui",
+    "version": "3.27.9999",
```

Comment:
**question:** Why `.9999`?

Reply:
See <https://github.com/DefinitelyTyped/DefinitelyTyped#how-do-definitely-typed-package-versions-relate-to-versions-of-the-corresponding-library>.